### DISCUSSION TITLE
Don't show camera preview - only show scan area

### DISCUSSION THREAD

**Discussion by GH20223** [juliansteenbakker/mobile_scanner#1063] [Q&A]
Hi,

I'm wondering if it is possible to hide the camera preview - or at least make it transparent or the size of the scan area. I would like to have only the 200x200 pixel window with the live preview and everything else should be purple.

I've tried to clip the MobileScanner widget or set explicit width and height with a surrounding container but then the scanner does not recognize any qr code anymore.

In the screenshot the "red area" should no longer be visible.

 
![scanner](https://github.com/juliansteenbakker/mobile_scanner/assets/115704197/b237bec7-95ff-400b-abb1-e1bdb112cc68)

**Comment by aalou-genetec** [comment]
I suggest putting the mobile scanner into a SizedBox and making the SizedBox the size of your scan area.
